### PR TITLE
Update terraform to version 0.12.26

### PR DESCRIPTION
**Pull request by tlatsas**
Updates terraform to version 0.12.26

- [x] Updated the versions and releases in the scraper file
- [x] Ensured the license is up-to-date and that the documentation's entry in the array in `about_tmpl.coffee` matches it's data in `self.attribution`
- [x] Ensured the icons and the `SOURCE` file in <code>public/icons/*your_scraper_name*/</code> are up-to-date if the documentation has a custom icon
- [x] Ensured `self.links` contains up-to-date urls if `self.links` is defined
- [x] Tested the changes locally to ensure:
  - The scraper still works without errors
  - The scraped documentation still looks consistent with the rest of DevDocs
  - The categorization of entries is still good

### CODE REVIEW

**Comment by tlatsas** [comment]
#1079

**Comment by simon04** [comment]
I'm checking a few pages and noticed that the updated docs no longer contain …
- https://devdocs.io/terraform/providers/dns/d/dns_aaaa_record_set
- https://devdocs.io/terraform/providers/icinga2/r/host
- https://devdocs.io/terraform/providers/postgresql/r/postgresql_database
- https://devdocs.io/terraform/providers/azurerm/r/image
- …

Is this intended?

File: lib/docs/scrapers/terraform.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Terraform < UrlScraper
 
     html_filters.push 'terraform/entries', 'terraform/clean_html'
 
-    options[:skip_patterns] = [/enterprise/, /enterprise-legacy/]
+    options[:skip_patterns] = [/enterprise/, /enterprise-legacy/, /guides/]
 
     options[:attribution] = <<-HTML
       &copy; 2018 HashiCorp</br>
```
Comment (simon04) [inline lib/docs/scrapers/terraform.rb:18]: The attribution string is outdated.

**Comment by tlatsas** [comment]
Hi @simon04! I believe at some point between the time I opened this PR and now, the docs pages have gone through a redesign so I should probably rework this a bit.

**Pull request closed without merging**